From 9d0f6411573cd5495346ed71583603fa3f8cd694 Mon Sep 17 00:00:00 2001
From: Alejandro Sierra <b18039@freescale.com>
Date: Thu, 14 Jan 2016 14:22:29 -0600
Subject: [PATCH 10/29] imx: mx6dqscm: qwks board dtb file name updated

QWKS dtb file on defconfig updated to support rev2 boards.

Signed-off-by: Alejandro Sierra <b18039@freescale.com>
---
 configs/mx6dqscm_1gb_interleaving_qwks_defconfig        | 2 +-
 configs/mx6dqscm_1gb_interleaving_qwks_spinor_defconfig | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/configs/mx6dqscm_1gb_interleaving_qwks_defconfig b/configs/mx6dqscm_1gb_interleaving_qwks_defconfig
index 894734b..d1f403f 100644
--- a/configs/mx6dqscm_1gb_interleaving_qwks_defconfig
+++ b/configs/mx6dqscm_1gb_interleaving_qwks_defconfig
@@ -1,4 +1,4 @@
-CONFIG_SYS_EXTRA_OPTIONS="IMX_CONFIG=board/freescale/mx6dqscmqwks/imximage_scm_lpddr2.cfg,MX6Q,DDR_MB=1024,SYS_USE_SPINOR,DEFAULT_FDT_FILE=\"imx6dqscm-1gb-qwks-rev1-ldo.dtb\",INTERLEAVING_MODE"
+CONFIG_SYS_EXTRA_OPTIONS="IMX_CONFIG=board/freescale/mx6dqscmqwks/imximage_scm_lpddr2.cfg,MX6Q,DDR_MB=1024,SYS_USE_SPINOR,DEFAULT_FDT_FILE=\"imx6dqscm-1gb-qwks-rev2-ldo.dtb\",INTERLEAVING_MODE"
 CONFIG_ARM=y
 CONFIG_TARGET_MX6DQSCMQWKS=y
 CONFIG_DM=y
diff --git a/configs/mx6dqscm_1gb_interleaving_qwks_spinor_defconfig b/configs/mx6dqscm_1gb_interleaving_qwks_spinor_defconfig
index 58587a8..c677982 100644
--- a/configs/mx6dqscm_1gb_interleaving_qwks_spinor_defconfig
+++ b/configs/mx6dqscm_1gb_interleaving_qwks_spinor_defconfig
@@ -1,4 +1,4 @@
-CONFIG_SYS_EXTRA_OPTIONS="IMX_CONFIG=board/freescale/mx6dqscmqwks/imximage_scm_lpddr2.cfg,MX6Q,DDR_MB=1024,SYS_BOOT_SPINOR,DEFAULT_FDT_FILE=\"imx6dqscm-1gb-qwks-rev1-ldo.dtb\",INTERLEAVING_MODE"
+CONFIG_SYS_EXTRA_OPTIONS="IMX_CONFIG=board/freescale/mx6dqscmqwks/imximage_scm_lpddr2.cfg,MX6Q,DDR_MB=1024,SYS_BOOT_SPINOR,DEFAULT_FDT_FILE=\"imx6dqscm-1gb-qwks-rev2-ldo.dtb\",INTERLEAVING_MODE"
 CONFIG_ARM=y
 CONFIG_TARGET_MX6DQSCMQWKS=y
 CONFIG_SYS_MALLOC_F=y
-- 
2.7.4

